src/test/SDL_test_harness.c

```diff
@@ -80,10 +80,8 @@ char *SDLTest_GenerateRunSeed(const int length)
     }
 
     /* Generate a random string of alphanumeric characters */
-    SDLTest_RandomInitTime(&randomContext);
     for (counter = 0; counter < length; counter++) {
-        unsigned int number = SDLTest_Random(&randomContext);
-        char ch = (char)(number % (91 - 48)) + 48;
+        char ch = (char)(SDL_rand_r(&randomContext, (91 - 48) + 1) + 48);
```

Comment:
I think the +1 is wrong. 91-48 = 43 = 26 (letters) + 10 (digits) + 7 (symbols)

rand_r(43) -> [0..42], +48 = [48-90]